modules/TimeSmearingNeutral.cc

```diff
@@ -0,0 +1,130 @@
+/*
```

Comment:
Why do we need a new module? 
Can't we just modify TimeSmearing to allow for both neutral and MIPs?
And then just call it in the card as: 

TimeSmearing TimeSmearingNeutral
TimeSmearing TimeSmearingMIP